cli/src/config.rs

```diff
@@ -281,9 +282,14 @@ impl ConfigEnv {
             repo_path: None,
             user_config_path: env.resolve()?,
             repo_config_path: ConfigPath::Unavailable,
+            command: None,
         })
     }
 
+    pub fn set_command(&mut self, command: Option<String>) {
```

Comment:
nits: 
* `set_command_name()` for clarity? (Perhaps, context variable can also be renamed.)
* just take `String`? It's unlikely that we need to clear the command name.